packages/eslint-plugin/src/rules/return-await.ts

```diff
@@ -271,92 +314,69 @@ export default createRule({
       const type = checker.getTypeAtLocation(child);
       const isThenable = tsutils.isThenableType(checker, expression, type);
```

Comment:
github's diffing gets real confused about what if you don't use the "ignore whitespace" here. Be sure to use that.